website_sale_secondary_unit/readme/USAGE.rst

```diff
@@ -0,0 +1,9 @@
+To use this module you need to:
+
+* Go to *'Sales > Catalog > Products'*.
```

Comment:
This seems to be outdated... and I think it is necessary to add a mention that "Units of Measure" needs to be enabled in the configuration.
Where is the menu to manage the secondary units?

Reply:
I don't know what menu do you mean 😅

Reply:
Hi @Tardo there is not a specific menu.
About the "Units of Measure" mention must be update in base module, not in this... but for OK...